src/Medulla/Kubernetes/Operator/Model/Database/ValidateSpec.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+namespace Medulla.Kubernetes.Operator.Model.Database;
+
+/// <summary>
+/// Spec in Column for validation
+/// </summary>
+public struct ValidateSpec
+{
+
+    /// <summary>
+    /// ValidateSpec Constructor
+    /// </summary>
+    public ValidateSpec()
```

Comment:
Can be deleted, no need for an empty constructor

Reply:
I believe @keithlogan94  added that there because we do need a constructor when we add default values. Please see the following error once I remove the empty constructor:

`
PS C:\Users\Baban\medulla\medulla\src\Medulla\Kubernetes\Operator> dotnet run
C:\Users\Baban\medulla\medulla\src\Medulla\Kubernetes\Operator\Model\Database\ValidateSpec.cs(10,15): error CS8983: A 'struct' with field initializers must include an explicitly declared constructor. [C:\Users\Baban\medulla\medulla\src\Medulla\Kubernetes\Operator\Operator.csproj]

The build failed. Fix the build errors and run again.
`

Reply:
ValidateSpec.cs(10,15): error CS8983: A 'struct' with field initializers must include an explicitly declared constructor.

Reply:
@AndrewBabbitt97 I added it because of this error

Reply:
I also followed the same pattern in the other files in this PR.

Reply:
I assume we can keep this right @AndrewBabbitt97  ?  If so, all other comments have been addressed too.